ui/src/app/components/help-sidebar.tsx

```diff
@@ -515,12 +541,12 @@ export const HelpSidebar = fp.flow(withCurrentWorkspace(), withUserProfile(), wi
       return <React.Fragment>
         <div style={notebookStyles ? {...styles.iconContainer, ...styles.notebookOverrides} : {...styles.iconContainer}}>
           {!criteria && this.renderWorkspaceMenu()}
-          {icons(helpContentKey).map((icon, i) =>
+          {icons(helpContentKey, serverConfigStore.getValue().enableCustomRuntimes, workspace.accessLevel).map((icon, i) =>
           this.showIcon(icon) && <div key={i} style={{display: 'table'}}>
                 <TooltipTrigger content={<div>{tooltipId === i && icon.tooltip}</div>} side='left'>
                   <div style={activeIcon === icon.id ? iconStyles.active : icon.disabled ? iconStyles.disabled : styles.icon}
                        onClick={() => {
-                         if (icon.id !== 'dataDictionary') {
+                         if (icon.id !== 'dataDictionary' && icon.disabled === false) {
```

Comment:
Was this a bug?

also nit: `!icon.disabled` seems more natural

Reply:
Yeah - it was opening on click even when disabled